packages/web-components/fast-components/src/menu-item/menu-item.ts

```diff
@@ -0,0 +1,57 @@
+import { attr, booleanConverter, FASTElement } from "@microsoft/fast-element";
+import { StartEnd } from "../patterns/start-end";
+import { applyMixins } from "../utilities/apply-mixins";
+
+export enum MenuItemRole {
+    menuitem = "menuitem",
+    menuitemcheckbox = "menuitemcheckbox",
+    menuitemradio = "menuitemradio",
+}
+
+export class MenuItem extends FASTElement {
+    @attr({ mode: "boolean" })
+    public disabled: boolean;
+    private disabledChanged(): void {
```

Comment:
Can you change this so that these classes are added in the template? Doing so batches the DOM changes and gets us some perf improvements.